cmd/kubeadm/app/phases/upgrade/staticpods.go

```diff
@@ -263,11 +264,18 @@ func performEtcdStaticPodUpgrade(client clientset.Interface, waiter apiclient.Wa
 	}
 
 	// gets the etcd version of the local/stacked etcd member running on the current machine
+	// the version is read from che cluster; this should take into account that there are still
+	// around old etcd manifest with etcd listening on local host only
+	etcdUrl := etcdutil.GetClientURL(&cfg.LocalAPIEndpoint)
+	if etcdutil.IsEtcdLinsteningOnLocalHostOnly() {
+		etcdUrl = "localhost:2379"
```

Comment:
do we need to support the localhost case for 1.14 and after?
i was thinking that we should mutate the manifest and cert during upgrade.

Reply:
That's only needed to fetch the etcd version number. For nothing else.

Reply:
The goal of this PR is to get rid of the localhost case in v1.14 (and it does this by replacing the manifest)